### PR TITLE
Add a scarecrow project to test GitHub issue filing/closing

### PR DESCRIPTION
**Pull request by Alan32Liu**
This scarecrow project intends to test `OSS-FUZZ`'s ability to file/close GitHub issues when a bug is detected/fixed.
Its `scarecrow.cc` imitates the [toy example from LLVM](https://llvm.org/docs/LibFuzzer.html#toy-example), which has a planted bug.

### CODE REVIEW

**Comment by Alan32Liu** [comment]
I did two tiny local experiments (with `scarecrow.cc` in this repo and in the project repo) to ensure it works.
I put it in the project directory in the end, just to have a relatively clearer isolation of implementation and separation of concern.

File: projects/github_scarecrow/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 Google Inc.
```
Comment (jonathanmetzman) [inline projects/github_scarecrow/Dockerfile:1]: nit: LLC not Inc.

File: projects/github_scarecrow/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (jonathanmetzman) [inline projects/github_scarecrow/Dockerfile:3]: is "scarecrow" a common term for this type of stuff? I've never heard of it before.
Maybe we can name it "github_filing_test" to be more descriptive?

File: projects/github_scarecrow/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update
```
Comment (jonathanmetzman) [inline projects/github_scarecrow/Dockerfile:18]: This seems unnecessary right? Let's delete it.

File: projects/github_scarecrow/build.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
+# Copyright 2022 Google Inc.
```
Comment (jonathanmetzman) [inline projects/github_scarecrow/build.sh:2]: nit: LLC

File: projects/github_scarecrow/Dockerfile (hunk 2 above)
Comment (jonathanmetzman) [reply projects/github_scarecrow/Dockerfile:3]: I guess you can leave the name as is because your repo is named scarecrow already.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Review by oliverchang — APPROVED** [review]

**Pull request merged**